scripts/corr2_dsim_control.py

```diff
@@ -131,7 +133,7 @@
         sinesource.set(scale=xscale, frequency=yfreq)
         print "Sine Source:", sinesource.name
         print "Scale:", sinesource.scale
-        print "Frequency:", sinesource.frequency        
+        print "Frequency:", sinesource.frequency
     except AttributeError:
```

Comment:
Also, it is better to have very small try: .. except: bodies. IN this case an AttributeError could potentially have been raised due to a bug in your print statements, but the program would act as if it was a user error. It would be better to have only the sinesource = getattr.... line in the try... except AttributeError block. If you want to catch other kinds of errors, make new try-except blocks.